graylog2-web-interface/src/pages/GrokPatternsPage.jsx

```diff
@@ -1,13 +1,15 @@
 import React from 'react';
 
 import GrokPatterns from 'components/grok-patterns/GrokPatterns';
-import { DocumentTitle } from 'components/common';
+import { DocumentTitle, IfPermitted } from 'components/common';
 
 const GrokPatternsPage = React.createClass({
   render() {
     return (
       <DocumentTitle title="Grok patterns">
-        <GrokPatterns />
+        <IfPermitted permissions="inputs:read">
+          <GrokPatterns />
```

Comment:
Shouldn't we leave the permissions check to the component that renders the entity? With your changes, we are already checking in `GrokPatterns` for the right permissions, so I guess this other check can be avoided.

Reply:
True!